client/app/scripts/superdesk-archive/module.js

```diff
@@ -140,8 +140,8 @@ define([
          * @param {Object} desk when passed the item will be assigned to this desk instead of user's activeDesk.
          */
         this.addTaskToArticle = function (item, desk) {
-            if ((!item.task || !item.task.desk) && (desk || desks.activeDeskId && desks.userDesks)) {
-                var currentDesk = desk || _.find(desks.userDesks._items, {_id: desks.activeDeskId});
+            if ((!item.task || !item.task.desk) && (desk || desks.getCurrentDeskId() && desks.userDesks)) {
```

Comment:
`getCurrentDeskId()` will always return `deskId` if user is member of desk(s). This prevents from creating articles in personal workspace. That's the reason `activeDeskId` was used.

If this is been taken care of then there should be test to check if creation of article is possible or not in personal workspace.